src/core/router/garlic.cc

```diff
@@ -245,13 +243,13 @@ std::size_t GarlicRoutingSession::CreateAESBlock(
   bool create_new_tags =
     m_Owner &&
     m_NumTags &&
-    (static_cast<int>(m_SessionTags.size()) <= m_NumTags * 2 / 3);
+    (static_cast<std::size_t>(m_SessionTags.size()) <= m_NumTags * 2 / 3);
```

Comment:
We can drop the `static_cast` now, `std::list` usually returns a `std::size_t` here.

Reply:
Good catch, thanks. Fixed.